nodemapper/src/gui/Builder/components/BodyWidget.tsx

```diff
@@ -90,9 +90,7 @@ export const BodyWidget = () => {
                   </div>
                 </Panel>
               </>
-            ) : (
-              <></>
-            )}
+            ) : null}
```

Comment:
Or just change `?` to `&&` and remove ` : null`

Reply:
Nice, thanks!